tests/integration/support/client_wrapper.go

```diff
@@ -313,3 +327,42 @@ func (c *ClientWrapper) track(request models.APIOptions) (models.APIResponse, er
 	response.Result = "NULL"
 	return response, err
 }
+
+func createUserProfileService(config pkg.ProjectConfig, apiOptions models.APIOptions) decision.UserProfileService {
```

Comment:
This doesn't need to be in the wrapper. We can make it helper method in `optlyplugins/user_profile_service` inside the `utils.go` file